test/systemtests/kubeadm_test.go

```diff
@@ -0,0 +1,981 @@
+	k.exec(podName, stopRestartCmd, "kube-system")
+
+	logrus.Infof("Stopping netplugin on %s", k.node.Name())
+	killNetpluginCmd := `pkill netplugin`
+	_, err = k.exec(podName, killNetpluginCmd, "kube-system")
+	return err
+}
+
+func (k *kubePod) stopNetmaster() error {
+	if k.node.Name() != "k8master" {
+		return nil
+	}
+	podName, err := getPodName("netmaster", k.node.Name())
+	if err != nil {
+		logrus.Errorf("pod not found: %+v", err)
+		return err
+	}
+
+	stopRestartCmd := `rm ` + netmasterRestartFile
+	k.exec(podName, stopRestartCmd, "kube-system")
+
+	logrus.Infof("Stopping netmaster on %s", k.node.Name())
+	killNetmasterCmd := `pkill netmaster`
+	_, err = k.exec(podName, killNetmasterCmd, "kube-system")
+	return err
+}
+
+func (k *kubePod) startNetmaster(args string) error {
+	if k.node.Name() != "k8master" {
+		return nil
+	}
+	logrus.Infof("Starting netmaster on %s", k.node.Name())
+	podName, err := getPodName("netmaster", k.node.Name())
+	if err != nil {
+		logrus.Errorf("pod not found: %+v", err)
+		return err
+	}
+
+	netmasterStartCmd := k.node.suite.basicInfo.BinPath + `/netmaster` + ` -cluster-store=` + k.node.suite.basicInfo.ClusterStore + ` -cluster-mode=kubernetes ` + args + ` > ` + netmasterLogLocation + ` 2>&1`
+
+	return k.podExecBG(podName, netmasterStartCmd, "kube-system")
+}
+
+func (k *kubePod) cleanupMaster() {
+	if k.node.Name() != "k8master" {
+		return
+	}
+	logrus.Infof("Cleaning up master on %s", k8sMaster.Name())
+	podName, err := getPodName("contiv-etcd", k.node.Name())
+	if err != nil {
+		logrus.Errorf("pod not found: %+v", err)
+		return
+	}
+	clusterStoreInfo := strings.Split(k.node.suite.basicInfo.ClusterStore, "//")
+	etcdIP := clusterStoreInfo[len(clusterStoreInfo)-1]
+	logrus.Infof("Cleaning out etcd info on %s", etcdIP)
+
+	k.podExec(podName, `etcdctl -C `+etcdIP+` rm --recursive /contiv`, "kube-system")
+	k.podExec(podName, `etcdctl -C `+etcdIP+` rm --recursive /contiv.io`, "kube-system")
+	k.podExec(podName, `etcdctl -C `+etcdIP+` rm --recursive /docker`, "kube-system")
+	k.podExec(podName, `etcdctl -C `+etcdIP+` rm --recursive /skydns`, "kube-system")
+}
+
+func getPodName(podRegex, nodeName string) (string, error) {
+	podNameCmd := `kubectl -n kube-system get pods -o wide | grep ` + podRegex + ` | grep ` + nodeName + ` | cut -d " " -f 1`
+	podName, err := k8sMaster.tbnode.RunCommandWithOutput(podNameCmd)
+	if err != nil {
+		logrus.Errorf("Couldn't fetch pod info on %s", nodeName)
+		return "", err
+	}
+	podName = strings.TrimSpace(podName)
+	return podName, nil
+}
+
+func (k *kubePod) cleanupSlave() {
+	if k.node.Name() == "k8master" {
+		return
+	}
+	logrus.Infof("Cleaning up slave on %s", k.node.Name())
+	podName, err := getPodName("netplugin", k.node.Name())
+	if err != nil {
+		logrus.Errorf("pod not found: %+v", err)
+		return
+	}
+
+	ovsCleanupCmd := `ovs-vsctl list-br | grep contiv | xargs -rt -I % ovs-vsctl del-br %`
+	_, err = k.podExec(podName, ovsCleanupCmd, "kube-system")
```

Comment:
If we're not checking `err` or printing any warning, there's no reason to assign it.  You could get rid of the `=` and everything to the left of it entirely